src/app/loadout-builder/processWorker/types.ts

```diff
@@ -92,8 +92,15 @@ export interface IntermediateProcessArmorSet {
   maxPower: number;
 }
 
-export interface ProcessModMetadata {
-  season: number;
-  tag: string;
+export interface ProcessMod {
   energyType: DestinyEnergyType;
+  investmentStats: DestinyItemInvestmentStatDefinition[];
```

Comment:
If possible you probably don't want to have the entire investment stat definition sent over the worker boundary - can you slice out a smaller set of data?

Reply:
Yeah I probably can. There are a couple of things I need but I will see if I can trim the whole definition down.